shared/chat/conversation/messages/message-popup/attachment/index.js

```diff
@@ -74,15 +74,10 @@ const AttachmentPopupMenu = (props: Props) => {
       onHidden={props.onHidden}
       closeOnSelect={true}
       position={props.position}
-      style={{...stylePopup, ...props.style}}
+      containerStyle={props.style}
```

Comment:
I made the props of FloatingMenu an exact type, and caught some errors where a `style` prop was being passed in to `FloatingMenu` instead of a `containerStyle` prop. Since these props weren't doing anything and the display of all the popups seemed correct, the best course of action seemed to be to delete all the styles and change the name if any props are passed down.